README.md

```diff
@@ -183,10 +187,18 @@ The easiest way to build Open Match is to use the Makefile. Before you can use t
 
 ```bash
```

Comment:
Separately, could you add a note to what need to be the steps for OS X?

Reply:
You basically have to download the dependencies from the websites. I'll add links to the websites.